cosmoz-tabbable-behavior.html

```diff
@@ -210,7 +210,56 @@
 
 		_normalizeValue: function (value) {
 			return this.attrForSelected ? value : isNaN(value) || value === null ? value : Number(value);
+		},
+
+		/**
+		* True if the current element is visible.
+		*/
+		get _isVisible() {
+			return Boolean(this.offsetWidth || this.offsetHeight);
+		},
+
+		_onDescendantIronResize(event) {
+			if (this._notifyingDescendant || !this._isVisible) {
+				event.stopPropagation();
+				return;
+			}
+			const ev = Polymer.dom(event),
+				item =  ev.localTarget,
+				index = this.items.indexOf(item);
+
+			if (index < 0 || !item.isSelected) {
+				event.stopPropagation();
+				return;
+			}
+
+			if (!Polymer.Settings.useShadow) {
```

Comment:
If redefining `_onDescendantIronResize`, maybe you should consider this PR https://github.com/PolymerElements/iron-resizable-behavior/pull/35